pkg/systemd/logs.scss

```diff
@@ -2,6 +2,11 @@
 @import "../lib/journal.css";
 @import "./system-global.scss";
 
+body {
+    // Ensure journal UI fills the entire page (and does not run over)
+    display: grid;
+}
+
 #journal {
     display: grid;
```

Comment:
Do we still need this as well then? (Intuitively, `display:grid` should be inherited by all `<body>` children?)

Reply:
Display: grid is never inherited. Immediate children participate in the grid, certainly, but are not grid elements themselves. 

#journal is a grid (with children specified in different ways) within a grid that only has one cell (to stretch the child to the content, which means the page, in this case).